webpack.config.js

```diff
@@ -113,6 +113,7 @@ module.exports = {
         // but it's sufficient to scope it at the chunk level
         new webpack.ProvidePlugin({
             videojs: 'videojs',
+            React: 'react',
```

Comment:
We can remove this provide plugin as we're explicitly importing `React` everywhere we need it now.

Reply:
cool, will do, thanks George